src/components/Edit/Edit.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react"
+import { updateDynamo } from "../PhotoAnalysis/utils"
+import { LabelType } from "../../../types/label"
+
+type Props = {
+  recordData: any
+}
+
+const Edit: React.FC<Props> = ({ recordData }) => {
+  const [labelsArray, setLabelsArray] = useState<LabelType[]>([])
+  // TODO store labels in a state
+  useEffect(() => {
+    setLabelsArray(JSON.parse(recordData.state.item.labels))
+  }, [])
+
+  const updateLabel =
+    (index: number) => (e: React.FormEvent<HTMLInputElement>) => {
+      const newArr = [...labelsArray]
+      newArr[index].name = e.currentTarget.value
+      setLabelsArray(newArr)
+    }
+
+  const commitChanges = () => {
+    updateDynamo(recordData.state.item.id, labelsArray)
+  }
+
+  return (
+    <>
+      {labelsArray.map((label: LabelType, index: number) => {
+        return (
+          <div key={index} className="flex is-flex-direction-row">
```

Comment:
Don't forget to change this to a span with the class that formats it as a label, I think the class is `tag`. Also I think you should wrap the map in a div with the class `tags are-medium` -

Reply:
I did try this, but it doesn't seem to format the inputs as labels, or maybe I misunderstood

Reply:
It won't format it as a label but as a tag. Maybe I don't understand the issue?